kubetest/main.go

```diff
@@ -168,7 +169,8 @@ func defineFlags() *options {
 	flag.StringVar(&o.nodeTestArgs, "node-test-args", "", "Test args specifically for node e2e tests.")
 	flag.BoolVar(&o.noAllowDup, "no-allow-dup", false, "if set --allow-dup will not be passed to push-build and --stage will error if the build already exists on the gcs path")
 	flag.BoolVar(&o.nodeTests, "node-tests", false, "If true, run node-e2e tests.")
-	flag.BoolVar(&o.perfTests, "perf-tests", false, "If true, run tests from perf-tests repo.")
+	flag.StringVar(&o.perfTests, "perf-tests", "", "If not empty, runs tests from perf-tests repo using run-e2e.sh from the main directory with pertfTests value as a first parameter.")
```

Comment:
nit: Could you rename this to perf-test (both flag and variable)? (as it's single test)